core/commands/object/patch.go

```diff
@@ -287,9 +287,7 @@ to a file containing 'bar', and returns the hash of the new object.
 
 		var createfunc func() *dag.Node
 		if create {
-			createfunc = func() *dag.Node {
-				return &dag.Node{Data: ft.FolderPBData()}
-			}
+			createfunc = ft.EmptyDirNode
```

Comment:
- it seems this is making a unixfs dir the default intermediary / create function.
- this is probably not what we want for all patch ops.